docs/1.guide/6.cache.md

```diff
@@ -65,6 +65,8 @@ The stars will be cached in development inside ```.nitro/cache/functions/ghStars
 {"expires":1677851092249,"value":43991,"mtime":1677847492540,"integrity":"ZUHcsxCWEH"}
 ```
 
+::important Because the cached data is serialized to JSON, it is important that the cached function does not return anything that cannot be serialized, such as Symbols, Maps, Sets… ::
```

Comment:
This is valid BTW for **any** response. Exception / current limitation with cache vs normal routes is that it cannot handle _binary_ and _streams_